src/NuGetTrends.Web/appsettings.json

```diff
@@ -7,7 +7,6 @@
     ]
   },
   "Sentry": {
-    "TracesSampleRate": 1.0,
```

Comment:
Using the callback to filter out `POST /t`
Simple: `Name != "POST /t"` didn't really work: https://github.com/getsentry/sentry-dotnet/issues/2442